### PR TITLE
NEW - PDF pos sign database

### PR DESCRIPTION
**Pull request by BB2A-Anthony**
# NEW|New [*PDF pos sign database*]
[*Long description*]

#24044 PR for Increment to propal

Save pos to print sign for onligne sign

Pages if other documents are merged after like pdf product
posX
posY
Height
Width

### CODE REVIEW

**Comment by BB2A-Anthony** [comment]
@eldy Up for 18 please

File: htdocs/install/mysql/migration/17.0.0-18.0.0.sql [hunk 1]
```diff
@@ -83,6 +83,9 @@ ALTER TABLE llx_bank_account ADD COLUMN owner_country_id integer DEFAULT NULL;
 
 ALTER TABLE llx_prelevement_bons ADD COLUMN fk_bank_account integer DEFAULT NULL;
 
+ALTER TABLE llx_propal ADD COLUMN model_pdf_pos_sign VARCHAR(10) DEFAULT NULL AFTER last_main_doc;
```
Comment (eldy) [inline htdocs/install/mysql/migration/17.0.0-18.0.0.sql:86]: Can you provide an example of value you plzn to store in this field ?

**Comment by BB2A-Anthony** [comment]
It's value 25:10:15:30

posX:posY:Height:Width

File: htdocs/install/mysql/migration/17.0.0-18.0.0.sql (hunk 1 above)
Comment (BB2A-Anthony) [reply htdocs/install/mysql/migration/17.0.0-18.0.0.sql:86]: It's value 25:10:15:30

posX:posY:Height:Width

**Pull request closed without merging**